src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.ServiceModel.Duplex.builds" />
```

Comment:
You should harvest whatever you're old assembly was.  It is supposed to use the stable version: that's the whole point of harvesting.  I guessed that it matched the ref but it'll actually match whatever you were actually targeting (probably the same thing the src project was targeting).  Look inside the 4.3.0 package to see what's there.  You'll also see log spew around this if you check the msbuild log.